elkm1_lib/elk.py

```diff
@@ -25,9 +25,12 @@ def __init__(self, config, loop=None):
         self.connection_lost_callbk = None
         self._connection_retry_timer = 1
         self._message_decode = MessageDecode()
-
+        self._sync_handlers = []
+        self._descriptions_in_prog = {}
```

Comment:
Would be nicer to use full words. Use `_descriptions_in_progress` (`prog` is ambiguous)